packages/gossip/manager.go

```diff
@@ -146,8 +132,9 @@ func (m *Manager) SendTransaction(txData []byte, to ...identity.ID) {
 }
 
 func (m *Manager) GetAllNeighbors() []*Neighbor {
```

Comment:
Missing function comment. Consider renaming to `AllNeighbors()`

Reply:
done